### DISCUSSION TITLE
when compile - 'NoneType' object has no attribute 'get'

### DISCUSSION THREAD

**Discussion by ch-a11y** [smartcontractkit/full-blockchain-solidity-course-py#325] [Q&A] [answered]
I got stopped at this step. How to fix it? What am I missing
<img width="539" alt="Screen Shot 2021-10-15 at 6 49 53 AM" src="https://user-images.githubusercontent.com/81821340/140713242-a25c8b1a-dc1a-4e61-9549-7e15827b3d9d.png">
?

**Comment by cromewar** [comment, marked as answer]
Hello @ch-a11y its seems to be a problem with your brownie/ganache installations, for example if you installed them as super user (using sudo), another user faced the same problem here: https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/295
By them recommendation you should follow this steps:


- use virtual env,
- install brownie to the venv,
- reinstall your ganache-cli to the directory you own (or simply chown your node_modules dir)

**Reply by Anu-srinivasan** [reply]
Hi
I have similar error with respect to key:  The error is given below:
I am able to run in local ganache. But when I run "brownie run scripts/deploy.py --network rinkeby", the following error occurs.  Please help.

  File "C:\Users\RBI\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\_cli\run.py", line 49, in main
    return_value, frame = run(
  File "C:\Users\RBI\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File ".\scripts\deploy.py", line 29, in main
    deploy_simple_storage()
  File ".\scripts\deploy.py", line 8, in deploy_simple_storage
    account = get_account()
  File ".\scripts\deploy.py", line 26, in get_account
    return  accounts.add(config["wallets"]["from_key"])
TypeError: 'NoneType' object is not subscriptable

**Reply by cromewar** [reply]
None type means your key is not on your path, don't forget to add it manually if brownie does not recognize it.
